mkdocs.yml

```diff
@@ -60,4 +60,4 @@ nav:
     - Getting help: getting_help.md
 edit_uri: edit/main/docs/
 extra:
-    latest_version: 0.14.0
+    latest_version: 0.15.0
```

Comment:
I've double checked that this value is not affecting the web UI

<img width="1725" alt="Screenshot 2022-10-24 at 11 34 42" src="https://user-images.githubusercontent.com/951580/197496076-bbddc777-a197-4877-8904-5dc490161d76.png">